src/app/screens/LNURLPay.tsx

```diff
@@ -73,16 +74,34 @@ function LNURLPay(props: Props) {
       if (response.userName) {
         setUserName(response.userName);
       }
+      if (response.userEmail) {
+        setUserEmail(response.userEmail);
+      }
     });
   }, []);
 
+  const getPayerData = (details: LNURLPayServiceResponse) => {
+    if (
+      userName &&
+      userEmail &&
+      userName.length &&
```

Comment:
Does i.e. `userName?.length` get rid of the first two entries?  
Same question for situations further down.

Reply:
Yeah good point I suppose just having userName.length and userEmail.length would accomplish the same check